backend/src/main/java/org/polsl/backend/service/HardwareService.java

```diff
@@ -20,8 +20,8 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.time.LocalDateTime;
+import java.util.*;
```

Comment:
Prosiłbym, żeby nie korzystać z grupowych importów (to te z gwiazdką). Środowisko same takie tworzy, ale można to wyłączyć w ustawieniach:
Settings -> Editor -> Code style -> Java -> Imports -> Class count to use import with '*' -> ustaw na dużo, np 100
😎